focus-new-window@mdpenguin.org/README.md

```diff
@@ -0,0 +1,18 @@
+# Focus New Window
+
+GNOME's handling of window focus to prevent focus-stealing has caused a lot of programs to open windows in the background rather than having them open in the foreground, focused and ready to be used ([bug report](https://gitlab.gnome.org/GNOME/mutter/-/issues/673)).  Unfortunately, this issue made its way into Cinnamon and affects a lot of commonly used programs, such as the file picker from VSCode.  This extension overrides the way that Cinnamon handles windows that are requesting attention so that they will be raised to the foreground and focused.
+
+![Preview](./screenshot.png)
```

Comment:
The screenshot shows up correctly in GitHub in my fork and the other extensions that link images this way also show up correctly.  I've added a commit to remove the alt text so that it won't show up in the Spices page, though.  Will that do?

Reply:
I normally find out real time when it comes to the Spices page (unless there's a working example elsewhere). 😅 

What I meant though, was that you could use the absolute link that will be created once this is merged, i.e. `https://github.com/linuxmint/cinnamon-spices-extensions/blob/master/focus-new-window@mdpengiun.org/screenshot.png` (assuming the UUID stays what it currently is). That way, it renders on both sites/platforms (and elsewhere too).

Reply:
Let's see how it shows up.  I wouldn't want the screenshot to show up twice on the Spices page, which I think would happen based on what I'm seeing with other extensions.